std_msgs/Header header
  uint32 seq
  time stamp
  string frame_id
dingo_control/Angle FL_foot
dingo_control/Angle FR_foot
dingo_control/Angle RL_foot
dingo_control/Angle RR_foot